### PR TITLE
search: add function to return list of dicts from ES hits

### PR DESCRIPTION
**Pull request by monaawi**
* INSPIR-3375

### CODE REVIEW

File: backend/inspirehep/records/marshmallow/institutions/ui.py [hunk 1]
```diff
@@ -84,9 +84,7 @@ def get_subsidiary_institutions(data):
             .query(query)
             .params(size=10000, _source=["legacy_ICN", "control_number"])
         )
-        results = [
-            query_result.to_dict() for query_result in query_results.execute().hits
-        ]
+        results = InspireSearch.get_dict_results_from_hits(query_results.execute().hits)
```
Comment (michamos) [inline backend/inspirehep/records/marshmallow/institutions/ui.py:87]: isn't an `InstitutionSearch` already an `InspireSearch`?

File: backend/inspirehep/records/views.py [hunk 2]
```diff
@@ -44,7 +44,9 @@ def get(self, pid, record):
 
         citing_records_results = LiteratureSearch.citations(record, page, size)
         citing_records_count = citing_records_results.total["value"]
-        citing_records = [citation.to_dict() for citation in citing_records_results]
+        citing_records = InspireSearch.get_dict_results_from_hits(
```
Comment (michamos) [inline backend/inspirehep/records/views.py:47]: same question

**Comment by pazembrz** [comment]
I think I would put it in some helper, as now it's counter intuitive to call static method from class which we already used before. Or change `citations` method to return data already processed by `get_dict_results_from_hits`

**Comment by monaawi** [comment]
I can't call it inside the citations because I also need access to the ```Hit``` to get the total of the results. I think it's the same thing a helper or inside the search, maybe it is better inside the search as we seem to have other helper functions there too

**Pull request closed without merging**